src/OAuth2/Encryption/EncryptionInterface.php

```diff
@@ -4,8 +4,8 @@
 
 interface EncryptionInterface
 {
-    public function encode($payload, $key, $algorithm = null);
-    public function decode($payload, $key, $algorithm = null);
+    public function encode($payload, $key, $algorithm = null, $keyId = null);
```

Comment:
They `keyId` is a new feature to selecting a key from a list to match against.

see https://github.com/firebase/php-jwt/blob/master/Authentication/JWT.php#L119 and https://github.com/firebase/php-jwt/blob/master/Authentication/JWT.php#L119